docsrc/abstraction-alternative.fsx

```diff
@@ -1,7 +1,7 @@
 (*** hide ***)
 // This block of code is omitted in the generated HTML documentation. Use 
 // it to define helpers that you do not want to show in the documentation.
-#I "../../bin"
+#I "../bin"
```

Comment:
You can specify input to the tool in order to have another folder than the default.